curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/intermediate-strings.md

```diff
@@ -3,6 +3,10 @@ id: 5e7b9f070b6c005b0e76f061
 title: Intermediate Strings
 challengeType: 11
 videoId: KgT_fYLXnyk
+videoLocaleIds:
+  espanol: 7jjoyy7_RCk
+  italian: 44LA7i_f4ug
+  portuguese: dQw4w9WgXcQ
```

Comment:
Ha..ha.. very funny :bemused:

I am not sure I can approve this:
![image](https://user-images.githubusercontent.com/51722130/133908494-a34e4578-57fe-4874-a733-8a8b897ebdb5.png)